ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -165,6 +165,9 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
         }
     }
 
+    /// The accessibility label that should be applied for the back button.
```

Comment:
In the spirit of not wanting this to live on - probably should add in a comment that consumers should ideally not use this - this is just temporary to avoid duplicating resources until we bring better SwiftUI support into the repo such that SwiftUI clients don't need to reconstruct fluent elements